src/app/shared/mastodon/mastodon.component.html

```diff
@@ -68,11 +68,7 @@
             <div [innerHTML]="data.postContent"></div>
           </div>
           <div class="toot-media" *ngIf="data.mediaContent.length > 0">
-            <img
-              *ngFor="let picid of data.mediaContent"
-              src="{{ data.mediaContent[picid].preview_url }}"
-              alt="{{ data.mediaContent[picid].description }}"
-            />
+            <img *ngFor="let picid of data.mediaContent" src="{{ picid.preview_url }}" alt="{{ picid.description }}" />
```

Comment:
This fix implies `mediaContent` is a set/list of media items.  If so, would be good to rename `picid` to something more accurate.